### ISSUE TITLE
Packaged versions don't support trailscraper guess

### ISSUE THREAD

**Issue by flosell** [flosell/trailscraper#110]
Looks like `known-iam-actions.txt` is not distributed in homebrew (introduced in #105) and pip: 

```
$ echo '{
   "Version":"2012-10-17",
   "Statement":[
      {
         "Effect":"Allow",
         "Action":[
            "s3:GetObject"
         ],
         "Resource":"arn:aws:s3:::my_corporate_bucket/home/${aws:userid}/*"
      }
   ]
}' | trailscraper guess
[...]
  File "/usr/local/Cellar/trailscraper/0.5.1/libexec/lib/python3.8/site-packages/trailscraper/iam.py", line 176, in all_known_iam_permissions
    with open(os.path.join(os.path.dirname(__file__), 'known-iam-actions.txt')) as iam_file:
FileNotFoundError: [Errno 2] No such file or directory: '/usr/local/Cellar/trailscraper/0.5.1/libexec/lib/python3.8/site-packages/trailscraper/known-iam-actions.txt'
```

**Comment by flosell** [comment]
Fixed in [latest release 0.6.1](https://github.com/flosell/trailscraper/releases/tag/0.6.1) in pip. 
Homebrew not released yet.

**Comment by flosell** [comment]
Homebrew released as well, fixed.
